train-winrm.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
 
   # Do not list inspec as a dependency of a train plugin.
   # Do not list train or train-core as a dependency of a train plugin.
-  spec.add_dependency "chef-winrm", ">= 2.3.11"
-  spec.add_dependency "chef-winrm-elevated", ">= 1.2.5"
-  spec.add_dependency "chef-winrm-fs", ">= 1.3.7"
+  spec.add_dependency "chef-winrm", "~> 2.3.12" # This version is required to support Ruby 3.0
```

Comment:
You specifically don't want to allow major versions?

Reply:
Yes for this version of train-winrm we want it to be restrictive so that it can have support for Ruby 3.0. Since `chef-winrm` also have support for Ruby 3.0 in `2.3.12` only with all the possible fixes.. In future releases, when we drop Ruby 3.0 support, we can change this.